docs/platforms/unreal/configuration/setup-crashreporter/index.mdx

```diff
@@ -182,3 +182,32 @@ Check out this community-generated [thread in Sentry's forum](https://forum.sent
 <Alert>☝ This feature is supported on Windows, Linux, and Android.</Alert>
 
 <Include name="store-minidumps-as-attachments-configuration" />
+
+## Sentry SDK and UE Crash Reporter feature comparison
```

Comment:
I think this should live up higher in the page. 

Since I believe folks will just get started with adding it before they even realize this.

Reply:
Good point, I've moved this to the top.